ci/tests/arakoon/extend_promote_demote_test.py

```diff
@@ -122,108 +143,74 @@ def teardown():
             cmd = 'rm -rf {0}'.format(location)
             general.execute_command_on_node(ip, cmd)
 
+    for key in KEY_CLEANUP:
+        cmd = 'etcdctl rm /ovs/arakoon/{0} --recursive'.format(key)
```

Comment:
Please use the extension
This is a general remark, more occurrences in other files can and should be replaced
Extension can be found here: /opt/OpenvStorage/ovs/extensions/db/etcd/configuration/EtcdConfiguration